TriblerGUI/widgets/trustpage/js/radial_view/radial_node.js

```diff
@@ -127,6 +127,15 @@ function RadialNodes(svg, options) {
     };
 
     /**
+     * Get the name of the node, either the public key or a user name.
```

Comment:
`a user name` seems quite vague. You could use a more descriptive documentation string such as `a string representing the node of the user himself`.